src/app/view/components/login/login-component/login-container.component.ts

```diff
@@ -44,30 +47,24 @@ export class LoginContainerComponent {
 
     logInUser(loginData: LoginData): void {
         this.isLoginInProgress = true;
```

Comment:
This need to be clean up because it's not being used now.

Reply:
this is still used, to disable input fields while doing the login.